q2_quality_control/tests/test_decontam.py

```diff
@@ -115,6 +115,63 @@ def test_combined(self):
                 expecter_table = biom.Table.from_tsv(th, None, None, None)
             self.assertEqual(test_table, expecter_table)
 
+    def test_freq_prev_control_col_error_raised(self):
+        with self.assertRaises(ValueError) as context:
+            decontam_identify(
+                table=self.asv_table,
+                metadata=self.metadata_input,
+                method='frequency',
+                freq_concentration_column='quant_reading',
+                prev_control_column='Sample_or_Control')
+        self.assertIn("--prev-control-column", str(context.exception))
```

Comment:
Because the string `"--prev-control-column"` occurs in more than one error message it doesn't uniquely identify any error. You should choose the asserted substring such that it uniquely identifies the error being tested.

Reply:
I won't leave separate comments for the other tests, but make to sure to update them in this way as well.

Reply:
addressed in commit addressed in commit 82acad4b955d831983ba4ee95b4450d0f1109e4b